docs/Installation_and_Setup.md

```diff
@@ -95,6 +91,6 @@ You can paste the map input file to this location:
 ```
 
 ## Security and Passwords
-V2X Hub is middleware that runs on Linux Ubuntu 16.04 LTS or Ubuntu 18.04 LTS with future versions on Ubuntu 18.04 LTS. It is recommended that appropriate security and firewall settings be used on the computer running Linux, including conforming to your agency's security best practices and IT protocols.
+V2X Hub is middleware that runs on Linux Ubuntu within a Docker image.  It is recommended that appropriate security and firewall settings be used on the computer running Linux, including conforming to your agency's security best practices and IT protocols.

```

Comment:
While you're changing this line please also change `Linux Ubuntu` to `Ubuntu Linux`

Reply:
Updated